client/my-sites/earn/home.tsx

```diff
@@ -153,20 +158,33 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 	 * @returns {object} Object with props to render a PromoCard.
 	 */
 	const getReferralsCard = () => {
+		const isJetpackNotATSite: boolean = isJetpack && ! isATSite;
 		const cta = {
 			text: translate( 'Earn Cash from Referrals' ),
-			action: { url: 'https://refer.wordpress.com/', onClick: () => trackCtaButton( 'referral' ) },
+			action: {
+				url: isJetpackNotATSite
+					? 'https://jetpack.com/for/affiliates/'
+					: 'https://refer.wordpress.com/',
+				onClick: () => trackCtaButton( 'referral' ),
```

Comment:
Would you want to have different event name for different CTAs (Jetpack vs .com)?

Reply:
ah good catch!

Reply:
Oh I just remembered reading that data folks want tracks events not to be dynamically built:

>Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base.

via PCYsg-4S2-p2

Reply:
They're hardcoded

https://github.com/Automattic/wp-calypso/pull/35038/commits/ff311f65f9883cf656177fdba900a31897ec203d#diff-6d934f22e0b322d8ae8daa0978804e26R167

https://github.com/Automattic/wp-calypso/pull/35038/commits/ff311f65f9883cf656177fdba900a31897ec203d#diff-6d934f22e0b322d8ae8daa0978804e26R171